src/sidebar/components/share-annotations-panel.js

```diff
@@ -87,7 +87,7 @@ function ShareAnnotationsPanel({ analytics, flash }) {
               onClick={copyShareLink}
               title="copy share link"
               aria-label="Copy share link"
-              className="btn btn-clean share-annotations-panel__copy-btn"
+              className="share-annotations-panel__copy-btn"
```

Comment:
..for example, a base component could be called <IconButton> and it could take a `name` or better yet `src` variable for the icon. Perhaps even have a left or right alignment prop. Then it would take on all the scss stuff below w/o needing its own class (unless we needed to override something)

This clearly would be a todo-later-on task